bdk-swift/README.md

```diff
@@ -0,0 +1,49 @@
+# bdk-swift
```

Comment:
I think it would be good to add the list of supported targets (either their official triple name, or maybe just their more common name). I will open an issue to add this info to the other bindings' readmes as well. If I understand correctly, for Swift this would be:
```txt
aarch64-apple-ios
x86_64-apple-ios
aarch64-apple-ios-sim
aarch64-apple-darwin
x86_64-apple-darwin
```